src/libs/capture.h

```diff
@@ -39,7 +39,8 @@
 #define US_VIDEO_MAX_FPS		((uint)120)
 
 #define US_STANDARDS_STR		"PAL, NTSC, SECAM"
-#define US_FORMATS_STR			"YUYV, YVYU, UYVY, RGB565, RGB24, BGR24, MJPEG, JPEG"
+#define US_FORMATS_STR			"YUYV, YVYU, UYVY, YU12, YV12,"
+#define US_FORMATS2_STR         "RGB565, RGB24, BGR24, MJPEG, JPEG"
```

Comment:
Do it in one line, as it was before. I'll think about how to form this list beautifully automatically, but then.